### ISSUE TITLE
Better embedded messages for the advertising channel (images, formatting)

### ISSUE THREAD

**Issue by AdamsLT** [Southclaws/cj#185]
Currently it's pretty basic, which is fine. But it would be cool to let people set the thumbnail and do other formatting things for their messages (up to some arbitrary limit).

https://discordjs.guide/popular-topics/embeds.html
https://anidiots.guide/first-bot/using-embeds-in-messages

It would involve more logic to do everything and would need some preset rules but I think it would be a net benefit to people that are actually using that channel.

**Comment by Southclaws** [comment]
Yeah this would definitely be neat. I was thinking of some kind of format that users must adhere to. It'll be difficult to explain that to people. Probably best by making the bot DM the user.

**Comment by coool9** [comment]
This issue should be closed. @ADRFranklin
